src/index.js

```diff
@@ -2,10 +2,20 @@
 
 const debug = require('debug')
 const log = debug('libp2p:websockets')
-const SWS = require('simple-websocket')
+const SW = require('simple-websocket')
+const isNode = require('detect-node')
+let SWS
+if (isNode) {
+  SWS = require('simple-websocket-server')
```

Comment:
Would be better, try adding `'simple-websocket-server': '{}'` to the `externals` list